cypress/integration/common/step_definitions/TemplatePaging.js

```diff
@@ -6,43 +6,43 @@ import {
 } from "../../../page-objects/template.po";
 
 When('I navigate to the last page', () => {
-  cy.get('ul.pager>li').eq(dataTableLastPage).click()
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableLastPage).click()
 })
 
 When('I navigate to the first page', () => {
-  cy.get('ul.pager>li').eq(dataTableFirstPage).click()
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableFirstPage).click()
 })
 
 When('I navigate to the next page', () => {
-  cy.get('ul.pager>li').eq(dataTableNextPage).click()
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableNextPage).click()
 })
 
 When('I navigate to the previous page', () => {
-  cy.get('ul.pager>li').eq(dataTablePreviousPage).click()
+  cy.get('datatable-pager>ul.pager>li').eq(dataTablePreviousPage).click()
 })
 
 When('I navigate to page two', () => {
-  cy.get('ul.pager>li').eq(3).click()
+  cy.get('datatable-pager>ul.pager>li').eq(3).click()
 })
 
 Then('The last page is displayed', () => {
-  cy.get('ul.pager>li').eq(dataTableLastPage).should('have.class', 'disabled')
-  cy.get('ul.pager>li').eq(dataTableFirstPage).should('not.have.class', 'disabled')
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableLastPage).should('have.class', 'disabled')
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableFirstPage).should('not.have.class', 'disabled')
 })
 
 Then('The first page is displayed', () => {
-  cy.get('ul.pager>li').eq(dataTableLastPage).should('not.have.class', 'disabled')
-  cy.get('ul.pager>li').eq(dataTableFirstPage).should('have.class', 'disabled')
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableLastPage).should('not.have.class', 'disabled')
+  cy.get('datatable-pager>ul.pager>li').eq(dataTableFirstPage).should('have.class', 'disabled')
```

Comment:
I would refactor those selectors using constants as they're used in several places, which would make future updates a lot easier (one place to edit instead of several, which is error-prone)

Reply:
Agreed.  Will make the change.